src/components/FilterPopup/FilterPopup.tsx

```diff
@@ -70,7 +70,7 @@ export const FilterPopup: React.FC<FilterPopupProps> = props => {
             viewName,
             widgetName: widget.name
         }
-        if (!props.value) {
+        if (!(props.value || props.value === false)) {
```

Comment:
It probably would be better to directly checked for `null` and `undefined` values here as inverted checked is not readable without the context.
Otherwise seems ok.

Reply:
Fixet to directly check for `null` and `undefined`